theme/layout.html

```diff
@@ -1,60 +1,46 @@
-{%- block doctype -%}
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
-  "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
-{%- endblock %}
-{%- set url_root = pathto('', 1) %}
-{# XXX necessary? #}
-{%- if url_root == '#' %}{% set url_root = '' %}{% endif %}
-
-{%- macro css() %}
-  <link href="{{ pathto('_static/css/reset-min.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/css/base.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/css/code.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/css/colors.css', 1) }}" rel="stylesheet" type="text/css" />
-  <link href="{{ pathto('_static/pygments.css', 1) }}" rel="stylesheet" type="text/css" />
-
-  {%- for cssfile in css_files %}
-    <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css" />
-  {%- endfor %}
-{%- endmacro %}
+<!DOCTYPE html>
+<html lang="{{ language|default('en') }}">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="description" content="{{ project }}">
+    {{ metatags }}
 
-{%- macro script() %}
-  <script>
-    var _gaq = _gaq || [];
-    _gaq.push(['_setAccount', 'UA-32006827-2']);
-    _gaq.push(['_trackPageview']);
+    <link href="{{ pathto('_static/css/reset-min.css', 1) }}" rel="stylesheet" type="text/css">
+    <link href="{{ pathto('_static/css/base.css', 1) }}" rel="stylesheet" type="text/css">
+    <link href="{{ pathto('_static/css/code.css', 1) }}" rel="stylesheet" type="text/css">
+    <link href="{{ pathto('_static/css/colors.css', 1) }}" rel="stylesheet" type="text/css">
+    <link href="{{ pathto('_static/pygments.css', 1) }}" rel="stylesheet" type="text/css">
 
-    (function() {
-      var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
-      ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
-      var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
-    })();
-  </script>
-{%- endmacro %}
+    {%- for cssfile in css_files %}
+      <link rel="stylesheet" href="{{ pathto(cssfile, 1) }}" type="text/css">
+    {%- endfor %}
 
-<html xmlns="http://www.w3.org/1999/xhtml"{% if language is not none %} lang="{{ language }}"{% endif %}>
-  <head>
-    <meta http-equiv="Content-Type" content="text/html; charset={{ encoding }}" />
-    <meta name="robots" content="index, follow, all" />
```

Comment:
should we keep this `robots` ?

Reply:
`all` already includes `Index, follow`...  and is the default value for the meta robots
https://developers.google.com/search/docs/crawling-indexing/robots-meta-tag .. 
So no impact at all there 

But i can guarantee you that the <title> tag fix will have one :)